### PR TITLE
Update And Test new version Flutter

### PR DESCRIPTION
**Pull request by yousifk**
merge with androidx
and update to 28 SDK

### CODE REVIEW

File: example/android/settings_aar.gradle [hunk 1]
```diff
@@ -0,0 +1 @@
+include ':app'
```
Comment (j0j00) [inline example/android/settings_aar.gradle]: Is this gradle file actually required?

**Review by j0j00 — COMMENTED** [review]
Thanks for the PR, really appreciate it. Could you clarify what the `settings_aar.gradle` file does?

File: example/android/settings_aar.gradle (hunk 1 above)
Comment (yousifk) [reply example/android/settings_aar.gradle]: I removed

**Comment by yousifk** [comment]
it takes a long time, why? @j0j00

**Comment by miguelcmedeiros** [comment]
It would be really great to get a new version with this PR included, otherwise we can’t use it with flutter 1.12.13. Looking forward to it 🙂

**Comment by rodruiz** [comment]
I'm using **flutter_user_agent: ^1.2.1** with flutter v1.12.13+hotfix.5 and no issues so far.
You can update the following in your own project:

File: PROJECT-DIR/android/gradle.properties
```properties
org.gradle.jvmargs=-Xmx1536M
android.useAndroidX=true
android.enableJetifier=true
android.enableR8=true
```

File: PROJECT-DIR/android/build.gradle
```gradle
...
dependencies {
    classpath 'com.android.tools.build:gradle:3.5.3'
}
```

File: PROJECT-DIR/android/app/build.gradle
```gradle
...
android {
    compileSdkVersion 29

....

    defaultConfig {
        minSdkVersion 21
        targetSdkVersion 29
        multiDexEnabled true
        versionCode flutterVersionCode.toInteger()
        versionName flutterVersionName
        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
    }
....
}
```

**Comment by j0j00** [comment]
Sorry @yousifk, I've been having PC problems so can't do anything at the moment. You should be able to work around it locally by targeting your fork in the `pubspec.yml`.

I think your code changes before the `examples` one were fine. The new one commits certain files which I don't think should be in version control, especially the ones which reference your local directory structure.

**Comment by j0j00** [comment]
Closing the PR for now until it's updated.

**Pull request closed without merging**